my-use-case/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
```

Comment:
general point for us to consider - poetry vs pipenv vs simple requirements.txt etc

Ideally not a mix!

Also we found that poetry does not work well (yet) with private PyPI mirrors (particularly auth to Google artifact registry). Upcoming poetry 1.2.0 release is expected to fix this though

Reply:
agreed, this is a tooling decision and should be used consistently. let's have a discussion! for now I'm aiming at a working example with poetry and tox.